src/components/new-request/name-input.vue

```diff
@@ -138,19 +133,18 @@ export default class NameInput extends Vue {
   async handleSubmit (event: KeyboardEvent) {
     if (event.key === 'Enter' && this.isCorpNumValid) {
       event.preventDefault()
-      if (this.searchValue) await this.analyzeName()
+      if (this.searchValue) {
+        // hide modal and perform name analysis
+        this.setMrasSearchInfoModalVisible(false)
+        if (this.getIsXproFlow) this.$root.$emit('showSpinner', true)
+        if (this.searchValue) await this.startAnalyzeName(null)
+        if (this.getIsXproFlow) this.$root.$emit('showSpinner', false)
+      }
       return
     }
     return event
   }
 
-  async analyzeName () {
```

Comment:
There was no advantage to having this as its own function so I moved the code above.